web/docs/project/env-vars.md

```diff
@@ -2,61 +2,165 @@
 title: Env Variables
 ---
 
+import ClientEnvVarsNote from './\_ClientEnvVarsNote.md'
+import { EnvVarsTable, EnvVar } from './EnvVarsTable'
+
 **Environment variables** are used to configure projects based on the context in which they run. This allows them to exhibit different behaviors in different environments, such as development, staging, or production.
 
-For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you may prefer it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
+For instance, _during development_, you may want your project to connect to a local development database running on your machine, but _in production_, you want it to connect to the production database. Similarly, in development, you may want to use a test Stripe account, while in production, your app should use a real Stripe account.
 
 While some env vars are required by Wasp, such as the database connection or secrets for social auth, you can also define your env vars for any other useful purposes, and then access them in the code.
 
-In Wasp, you can use environment variables in both the client and the server code.
+Let's go over the available env vars in Wasp, how to define them, and how to use them in your project.
+
 ## Client Env Vars
 
-Client environment variables are embedded into the client code during the build and shipping process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys -> you can provide those to server instead).
+Client environment variables are injected into the client Javascript code during the build process, making them public and readable by anyone. Therefore, you should **never store secrets in them** (such as secret API keys, you should store secrets in the server env variables).
 
-To enable Wasp to pick them up, client env vars must be prefixed with `REACT_APP_`, for example: `REACT_APP_SOME_VAR_NAME=...`.
+<ClientEnvVarsNote />
 
 You can read them from the client code like this:
 
 <Tabs groupId="js-ts">
 <TabItem value="js" label="JavaScript">
 
 ```js title="src/App.js"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 <TabItem value="ts" label="TypeScript">
 
 ```ts title="src/App.ts"
-console.log(import.meta.env.REACT_APP_SOME_VAR_NAME)
+import { env } from 'wasp/client'
+
+console.log(env.REACT_APP_SOME_VAR_NAME)
 ```
 </TabItem>
 </Tabs>
 
+Read more about the `env` object in the [API reference](#client-env-vars-1).
 
-Check below on how to define them.
+### Wasp Client Env Vars
 
+Here are the client env vars that Wasp defines:
+
+#### General Configuration
```

Comment:
We haven't introduced earlier this concept of "general configuration env vars" though, right? Should we, or it is self evident?

Reply:
This is where we introduce it :)

Reply:
Hm but why did I ask this then? I believe I asked it because you don't really explain it at any moment, you just go for it and start listing them. That is when I was reading it, I was thinking "oh ok general configuration, what is that" and then bam, vars immediately.
Which is why I asked above, should we explain what it is or you think it is self evident - when somebody sees "general configruation", they understand what these env vars are?

Reply:
I wrote this because when I was reading it I was confused for a bit, so I didn't make it up. Am I representative of a typical user, hard to say. Which is why I asked how you feel about it.

If I would write sometihng, I wouldn't write what you said because it is a bit much, especially the last part with "hence". I might write someting like "Env variables used by Wasp to configure general aspects of your app, not tied to one specific feature.".
But again, is that useful or not -> not sure. I believe I provided my perspective well enough here, I will leave the final decision to you.

Reply:
Added `These are some general env variables used for various Wasp features:` 👍